### DISCUSSION TITLE
Failed Upgrade

### DISCUSSION THREAD

**Discussion by fullsailj** [audacious-media-player/audacious#98] [General]
About a month ago  I tried to upgrade with APT as I normally would.  It failed as have all my attempts to recover.  I’ve run out of ideas and I’m giving in and asking for help (like I probably should have weeks ago).

I’m not sure what all would help, so here is a bit of a spaghetti approach.  Let me know if other information in needed.

One other thing this machine had linux mint installed 04/19/2020.  It went through a few upgrades to the OS, but failed on the last upgrade from 20.3 to 21.  I never got that sorted out.  I’ve been thinking of doing a clean install of linux mint 21.2.  I just dread moving all my data and loosing all my customizations (aplets, desklets, and other software).

`inxi -Fz
jim@crested-caracara:~ $ systeminfo 
------------------------------------------------------------------------------------------------------
-- crested-caracara system configuration as of: 01/05/2024 03:52:25 PM
------------------------------------------------------------------------------------------------------
System:    Kernel: 5.4.0-122-generic x86_64 bits: 64 Desktop: Cinnamon 5.2.7 
           Distro: Linux Mint 20.3 Una 
Machine:   Type: Desktop System: Dell product: OptiPlex 7010 v: 01 serial: <filter> 
           Mobo: Dell model: 0GY6Y8 v: A01 serial: <filter> BIOS: Dell v: A29 date: 06/28/2018 
CPU:       Topology: Quad Core model: Intel Core i5-3470 bits: 64 type: MCP L2 cache: 6144 KiB 
           Speed: 1670 MHz min/max: 1600/3600 MHz Core speeds (MHz): 1: 1597 2: 1656 3: 1597 
           4: 1616 
Graphics:  Device-1: Intel Xeon E3-1200 v2/3rd Gen Core processor Graphics driver: i915 v: kernel 
           Display: x11 server: X.Org 1.20.13 driver: modesetting unloaded: fbdev,vesa 
           resolution: 1680x1050~60Hz, 1680x1050~60Hz 
           OpenGL: renderer: Mesa DRI Intel HD Graphics 2500 (IVB GT1) v: 4.2 Mesa 21.2.6 
Audio:     Device-1: Intel 7 Series/C216 Family High Definition Audio driver: snd_hda_intel 
           Sound Server: ALSA v: k5.4.0-122-generic 
Network:   Device-1: Intel 82579LM Gigabit Network driver: e1000e 
           IF: eno1 state: up speed: 1000 Mbps duplex: full mac: <filter> 
Drives:    Local Storage: total: 1.47 TiB used: 1.29 TiB (87.6%) 
           ID-1: /dev/sda vendor: Kingston model: SV300S37A120G size: 111.79 GiB 
           ID-2: /dev/sdb vendor: Seagate model: ST31500341AS size: 1.36 TiB 
Partition: ID-1: / size: 109.47 GiB used: 50.32 GiB (46.0%) fs: ext4 dev: /dev/sda1 
Sensors:   System Temperatures: cpu: 42.0 C mobo: N/A 
           Fan Speeds (RPM): N/A 
Info:      Processes: 260 Uptime: 15h 10m Memory: 15.52 GiB used: 5.57 GiB (35.9%) Shell: bash 
           inxi: 3.0.38 
------------------------------------------------------------------------------------------------------

jim@crested-caracara:~ $ sudo apt-get install audacious
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 audacious : Depends: audacious-plugins (>= 4.3.1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

jim@crested-caracara:~ $ sudo apt update --fix-missing
Ign:1 http://mirror.cs.jmu.edu/pub/linuxmint/packages tricia InRelease
Hit:2 http://mirror.clarkson.edu/ubuntu bionic InRelease                                              
Hit:3 http://mirror.cs.jmu.edu/pub/linuxmint/packages tricia Release                                  
Hit:4 http://mirror.clarkson.edu/ubuntu bionic-updates InRelease                                      
Hit:5 http://mirror.clarkson.edu/ubuntu bionic-backports InRelease                                    
Hit:6 http://security.ubuntu.com/ubuntu bionic-security InRelease                                     
Hit:8 http://archive.canonical.com/ubuntu bionic InRelease                                       
Hit:9 http://ppa.launchpad.net/flacon/ppa/ubuntu bionic InRelease                          
Hit:10 http://ppa.launchpad.net/hamishmb/myppa/ubuntu bionic InRelease
Hit:11 http://ppa.launchpad.net/jonaski/strawberry/ubuntu focal InRelease
Hit:12 http://ppa.launchpad.net/jonathonf/vim/ubuntu bionic InRelease
Ign:13 https://apt.fury.io/ascii-image-converter  InRelease
Hit:14 http://ppa.launchpad.net/jonathonf/vim/ubuntu focal InRelease
Ign:15 https://apt.fury.io/ascii-image-converter  Release
Ign:16 https://apt.fury.io/ascii-image-converter  Packages
Ign:17 https://apt.fury.io/ascii-image-converter  Translation-en_US
Hit:18 http://ppa.launchpad.net/libreoffice/ppa/ubuntu bionic InRelease
Ign:19 https://apt.fury.io/ascii-image-converter  Translation-en
Get:16 https://apt.fury.io/ascii-image-converter  Packages
Hit:20 http://ppa.launchpad.net/lucioc/sayonara/ubuntu focal InRelease
Ign:17 https://apt.fury.io/ascii-image-converter  Translation-en_US   
Hit:21 http://ppa.launchpad.net/nilarimogard/webupd8/ubuntu bionic InRelease
Ign:19 https://apt.fury.io/ascii-image-converter  Translation-en
Ign:17 https://apt.fury.io/ascii-image-converter  Translation-en_US
Ign:19 https://apt.fury.io/ascii-image-converter  Translation-en
Hit:22 http://ppa.launchpad.net/otto-kesselgulasch/gimp/ubuntu bionic InRelease
Ign:17 https://apt.fury.io/ascii-image-converter  Translation-en_US
Ign:19 https://apt.fury.io/ascii-image-converter  Translation-en
Hit:23 http://ppa.launchpad.net/savoury1/graphics/ubuntu focal InRelease
Ign:17 https://apt.fury.io/ascii-image-converter  Translation-en_US
Ign:19 https://apt.fury.io/ascii-image-converter  Translation-en
Ign:17 https://apt.fury.io/ascii-image-converter  Translation-en_US
Hit:24 http://ppa.launchpad.net/savoury1/multimedia/ubuntu focal InRelease
Ign:19 https://apt.fury.io/ascii-image-converter  Translation-en
Ign:17 https://apt.fury.io/ascii-image-converter  Translation-en_US
Hit:25 http://ppa.launchpad.net/team-xbmc/ppa/ubuntu bionic InRelease
Ign:19 https://apt.fury.io/ascii-image-converter  Translation-en
Hit:26 http://ppa.launchpad.net/ubuntuhandbook1/apps/ubuntu focal InRelease
Fetched 15.4 kB in 2s (7,095 B/s)
Reading package lists... Done
Building dependency tree       
Reading state information... Done
17 packages can be upgraded. Run 'apt list --upgradable' to see them.

jim@crested-caracara:~ $ sudo apt list --upgradable
Listing... Done
ffmpegthumbnailer/focal 2.2.2+dfsg-1~20.04.sav0 amd64 [upgradable from: 2.1.1-0.2build2]
gstreamer1.0-libav/focal 1.22.8-0ubuntu1~20.04.sav0 amd64 [upgradable from: 1.16.2-2]
guvcview/focal 2.0.8-2~20.04.sav0 amd64 [upgradable from: 2.0.6+debian-1build1]
handbrake/focal 1.5.1+ds1-1~20.04.sav1 amd64 [upgradable from: 1.3.1+ds1-1build1]
libchromaprint1/focal 1.5.1-1~20.04.sav0 amd64 [upgradable from: 1.4.3-3build1]
libffmpegthumbnailer4v5/focal 2.2.2+dfsg-1~20.04.sav0 amd64 [upgradable from: 2.1.1-0.2build2]
libguvcview-2.0-2/focal 2.0.7-2-1~20.04.sav0 amd64 [upgradable from: 2.0.6+debian-1build1]
libimage-magick-q16-perl/focal 8:6.9.10.23+dfsg-2.1ubuntu11.9+20.04.sav0.1 amd64 [upgradable from: 8:6.9.10.23+dfsg-2.1ubuntu11.4]
librtmp1/focal 2.4+20151223.gitfa8646d.1-2build2~20.04.sav0 amd64 [upgradable from: 2.4+20151223.gitfa8646d.1-2build1]
melt/focal 7.22.0-1~20.04.sav0 amd64 [upgradable from: 6.20.0-2]
mpd/focal 0.23.14-1~20.04.sav0 amd64 [upgradable from: 0.21.20-1build1]
qmmp/focal 1.4.4-1build1~20.04.sav0 amd64 [upgradable from: 1.3.1-1.1]
ubuntu-advantage-tools/bionic-updates 30~18.04 amd64 [upgradable from: 27.9~20.04.1]
xscreensaver-data-extra/focal 6.08+dfsg1-0ubuntu1~20.04.sav0.1 amd64 [upgradable from: 6.06+dfsg1-3ubuntu0~20.04.sav0]
xscreensaver-data/focal 6.08+dfsg1-0ubuntu1~20.04.sav0.1 amd64 [upgradable from: 6.06+dfsg1-3ubuntu0~20.04.sav0]
xscreensaver-gl-extra/focal 6.08+dfsg1-0ubuntu1~20.04.sav0.1 amd64 [upgradable from: 6.06+dfsg1-3ubuntu0~20.04.sav0]
xscreensaver-gl/focal 6.08+dfsg1-0ubuntu1~20.04.sav0.1 amd64 [upgradable from: 6.06+dfsg1-3ubuntu0~20.04.sav0]
`

**Comment by jlindgren90** [comment]
That's a lot of PPAs.

Are you able to uninstall audacious? I would try that and then try the upgrade again.

**Reply by fullsailj** [reply]
Yes, I've tried (at least three times) to uninstall...
    `apt-get remove        and
    apt-get remove --purge`
When trying to subsequently install, the install fails in the same place as above.
